resource/inspire2019-prod/sidebarMetadata.json

```diff
@@ -224,52 +217,38 @@
         "subMetadata": []
       },
       {
-        "title": "Country",
+        "title": "Job Category",
```

Comment:
@gavinbarron  I noticed that there were some differences between our inspire2019-test and inspire2019-prod versions, so I moved over all the changes from inspire2019-test into prod as well.

Let me know if I wasn't supposed to do this, then I'll revert.

Reply:
I'd been holding off on moving these changes up to the prod rules until we're about to release into prod.

It's not a problem now, but it's often a good idea. I have broken prod before pushing an update that was a little off to both environments in the past.